ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -157,17 +159,22 @@ const Post = ({ post }: { post: PostType }) => {
 
   return (
     <div css={styles.postContainer}>
-      <PostHeaderBanner isBreakingNews={isBreakingNews} timestamp={timestamp} />
-      <div css={styles.postBody}>
+      <div css={styles.postBackground}>
         <Heading level={3}>
           {/* eslint-disable-next-line jsx-a11y/aria-role */}
           <span role="text">
+            <PostHeaderBanner
+              isBreakingNews={isBreakingNews}
+              timestamp={timestamp}
```

Comment:
I don't think you have specifally said output a zere before the number in the date/timestamp if it is a single digit e.g. '09', could we check with UX that's what we want please? Just '9' would be a better UX for users I would of thought? e.g. '9 May 2023, Breaking, Breaking news headline' rather than '09 May 2023, Breaking, Breaking news headline' It's more noticeable when there is no CSS.